pkg/infoschema/tables.go

```diff
@@ -854,6 +854,7 @@ var tableProcesslistCols = []columnInfo{
 	{name: "TxnStart", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag, deflt: ""},
 	{name: "RESOURCE_GROUP", tp: mysql.TypeVarchar, size: resourcegroup.MaxGroupNameLength, flag: mysql.NotNullFlag, deflt: ""},
 	{name: "SESSION_ALIAS", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag, deflt: ""},
+	{name: "AFFECTED_ROWS", tp: mysql.TypeLonglong, size: 21, flag: mysql.UnsignedFlag},
```

Comment:
Is there compatibility risks for tools like dashboard?

Reply:
I suppose they should not depend on the number of the columns...
@lcwangchao have you met any compatibility problems when adding SESSION_ALIAS?

Reply:
I think it's fine. The added row is in the last position. @bb7133 Could you take a look?

Reply:
I believe it is fine for compatibility. However, my concern is about the semantics when it is called 'affected rows': all columns in `PROCESSLIST` are the 'current status' but `AFFECTED_ROWS` means the affected rows by the last statement, which could be confusing.

For example, after executing `INSER T INTO ..`, we have:
```
            ID: 2097158
          USER: root
          HOST: 127.0.0.1:50816
            DB: test
       COMMAND: Sleep
          TIME: 16
         STATE: autocommit
          INFO: NULL
        DIGEST: 04fa858fa491c62d194faec2ab427261cc7998b3f1ccf8f6844febca504cb5e9
           MEM: 4096
          DISK: 0
      TxnStart:
RESOURCE_GROUP: default
 SESSION_ALIAS:
 AFFECTED_ROWS: 2
2 rows in set (0.01 sec)
```

Reply:
How about displaying `0` when `INFO` is null or `len(INFO) == 0`?

Reply:
I'm not sure, to be honest. Maybe we could also consider renaming it 'AFFECTED_ROWS_LAST_STATEMENT'?

What is the expected result of this column for pipelined DMLs?

Reply:
It's introduced to reveal the current progress of the statement during its execution, as a pipelined DML may run for a long time. So 'AFFECTED_ROWS_LAST_STATEMENT' is not a good idea I suppose.

Reply:
Or just rename it "CURRENT_AFFECTED_ROWS" to distinguish them and make it clear.

Reply:
Sounds good